sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -224,13 +227,15 @@ try:
         detected_language = translation.detected_language
         if detected_language:
             print(
-                f"Detected languages of the input text: {detected_language.language} with score: {detected_language.score}."
+                f"Detected languages of the input text: {detected_language.language} with score: {detected_language.confidence}."
             )
         for translated_text in translation.translations:
             print(f"Text was translated to: '{translated_text.to}' and the result is: '{translated_text.text}'.")
-            if translated_text.sent_len:
-                print(f"Source Sentence length: {translated_text.sent_len.src_sent_len}")
-                print(f"Translated Sentence length: {translated_text.sent_len.trans_sent_len}")
+            if translated_text.sentences_lengths:
+                print(f"Source Sentence length: {translated_text.sentences_lengths.src_sentences_lengths}")
+                print(
+                    f"Translated Sentence length: {translated_text.sentences_lengths.trans_sentences_lengths}"
```

Comment:
I think we should really use the full word `translated` for `translated_sentences_lengths` here

Reply:
Fixed.